imports/api/lib/graphql.js

```diff
@@ -0,0 +1,65 @@
+/**
+* Copyright 2019 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { Meteor } from 'meteor/meteor';
+
+const { ApolloLink } = require('apollo-link');
+const ApolloClient = require('apollo-boost').ApolloClient;
+const fetch = require('cross-fetch/polyfill').fetch;
+const createHttpLink = require('apollo-link-http').createHttpLink;
+const { onError } = require('apollo-link-error');
+const InMemoryCache = require('apollo-cache-inmemory').InMemoryCache;
+
+const getQueryClient = async () => {
+
+    const token = Meteor.user().apiKey ? Meteor.user().apiKey : Meteor.call('generateApikey');
```

Comment:
Doesnt the Meteor.call() need to have a callback and set it to a var. Something like:
```
let token = Meteor.user().apiKey
if(!token){
  token = await new Promise((resolve, reject)=>{
    Meteor.call('generateApikey', (err, result)=>{
      if(err){
        reject(err);
        return;
      }
      resolve(result);
    })
  })
}
```
Or maybe we can do `token = await bluebird.promisify(Meteor.call)('generateApikey')` to make that less verbose. (not sure if this code works)

Reply:
nevermind. it can be run sync